seaborn/utils.py

```diff
@@ -527,3 +527,29 @@ def categorical_order(values, order=None):
                     order = order
         order = filter(pd.notnull, order)
     return list(order)
+
+
+def to_unicode_or_bust(obj, encoding='utf-8'):
```

Comment:
"or_bust" suggests that an error will be raised if it fails, but I don't see that in the code. Maybe just "to_unicode" then?